### DISCUSSION TITLE
Can't add card usign method createCardToken

### DISCUSSION THREAD

**Discussion by FranciscoGarciaHorna** [mercadopago/sdk-js#82] [General]
When I create the token using the createCardToken method and I want to save the card, the service to save the card returns "Invalid credentials"
`const cardToken = await mp.createCardToken({
    cardNumber: '5031433215406351' ,
    cardholderName: 'APRO',
    expirationMonth: '11',
    expirationYear: '2025',
    securityCode: '123',
    identificationType: 'CPF',
    identificationNumber: '12345678912',
})
`

**Comment by lucmantovani** [comment]
Hello @FranciscoGarciaHorna!

This error might be related to some instability on our API. I will look into that and will let you know.

But I couldn't stop to notice that you are using what is now a legacy method of tokenization.

Recently we launched a new and more secure way of handling card data.
I recommend you to have a look at our [guide](https://www.mercadopago.com.ar/developers/es/docs/checkout-api/integration-configuration/card/integrate-via-core-methods) and to consider using this new way, unless you possess a specific use case.
[Here](https://github.com/mercadopago/sdk-js/blob/main/API/fields.md) you can find the method's description and usage 🙂
